src/service/url-replacements-impl.js

```diff
@@ -552,9 +566,19 @@ export class GlobalVariableSource extends VariableSource {
       return getter(/** @type {!ShareTrackingFragmentsDef} */ (fragments));
     });
   }
+
+    /**
+     * Format number to have 2 digits if number is smaller than 10
+     * @param num
+     * @returns {string} after formatted
+     * @private
+     */
+  formatNum_(num) {
+    return (num < 10 ? ('0' + num) : num);
```

Comment:
I think there are other uses who have something similar. A `leftPad` helper could be useful here.